diff --git a/capacitor/src/main/java/com/getcapacitor/WebViewLocalServer.java b/capacitor/src/main/java/com/getcapacitor/WebViewLocalServer.java
index a044bfbe6df53cc6ed962955dccd44886e1d003a..7565f66ff99bf770a909cdd83024ac2dfaf4c650 100755
--- a/capacitor/src/main/java/com/getcapacitor/WebViewLocalServer.java
+++ b/capacitor/src/main/java/com/getcapacitor/WebViewLocalServer.java
@@ -326,14 +326,14 @@ public class WebViewLocalServer {
     private WebResourceResponse handleLocalRequest(WebResourceRequest request, PathHandler handler) {
         String path = request.getUrl().getPath();
 
-        if (request.getRequestHeaders().get("Range") != null) {
+        if (request.getRequestHeaders().get("range") != null) {
             InputStream responseStream = new LollipopLazyInputStream(handler, request);
             String mimeType = getMimeType(path, responseStream);
             Map<String, String> tempResponseHeaders = handler.getResponseHeaders();
             int statusCode = 206;
             try {
                 int totalRange = responseStream.available();
-                String rangeString = request.getRequestHeaders().get("Range");
+                String rangeString = request.getRequestHeaders().get("range");
                 String[] parts = rangeString.split("=");
                 String[] streamParts = parts[1].split("-");
                 String fromRange = streamParts[0];
